operations/kubernetes/Repopulate_Data_in_etcd_Clusters_When_Rebuilding_Them.md

```diff
@@ -37,7 +37,7 @@ Reconstruct boot session templates for impacted product streams to repopulate da
 Boot preparation information for other product streams can be found in the following locations:
 
 - UANs: Refer to the UAN product stream repository and search for the "PREPARE UAN BOOT SESSION TEMPLATES" header in the "Install and Configure UANs" procedure.
-- Cray Operating System \(COS\): Refer to the "Create a Boot Session Template" header in the "Boot COS" procedure in the COS product stream documentation.
+- Cray Operating System \( (USS)\): Refer to the "Create a Boot Session Template" header in the "Boot  (USS)" procedure in the  (USS) product stream documentation.
```

Comment:
I don't think this section name exists anymore.  See https://hpcdc-web.hpc.amslabs.hpecorp.net/~pubs/USS_docs/master/CSM/build/USS_Documentation_for_CSM/pdf/USS_Administration_Guide_CSM_on_EX_Systems_S-8063.pdf.  Maybe a more generic reference is fine?